cypress.config.js

```diff
@@ -0,0 +1,14 @@
+const { defineConfig } = require("cypress");
```

Comment:
Que tal usar aspas simples aqui também, para manter o padrão usado em outras strings deste arquivo?

```suggestion
const { defineConfig } = require('cypress');
```

Reply:
alterado a aspas, conforme apontado.